shoulder_pitch_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: shoulder_pitch_joint
    joint_speed: 1.17
    motor:
        id: 1
        init: 2360
        min: 0
        max: 3340

shoulder_pan_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: shoulder_pan_joint
    joint_speed: 1.17
    motor:
        id: 2
        init: 3134
        min: 0
        max: 3980

upperarm_roll_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: upperarm_roll_joint
    joint_speed: 1.17
    motor:
        id: 3
        init: 588
        min: 0
        max: 4095

elbow_flex_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: elbow_flex_joint
    joint_speed: 1.17
    motor:
        id: 4
        init: 2368
        min: 823
        max: 4095

forearm_roll_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: forearm_roll_joint
    joint_speed: 1.17
    motor:
        id: 5
        init: 493
        min: 0
        max: 1023

wrist_pitch_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: wrist_pitch_joint
    joint_speed: 1.17
    motor:
        id: 6
        init: 512
        min: 90
        max: 1023

wrist_roll_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: wrist_roll_joint
    joint_speed: 1.17
    motor:
        id: 7
        init: 508
        min: 0
        max: 1023

